service/validation.go

```diff
@@ -71,13 +72,15 @@ func (v *parameterValidator) validateType(value interface{}) []*ParameterWarning
 		}
 
 	case "Number":
-		_, okFloat64 := value.(float64)
-		_, okFloat32 := value.(float32)
-		_, okInt := value.(int)
-		if !okInt && !okFloat64 && !okFloat32 {
+		switch value.(type) {
+		case
+			uint8, uint16, uint32, uint64, uint,
+			int8, int16, int32, int64, int,
+			float64, float32,
+			*big.Int, *big.Float:
```

Comment:
Why having `*big.Int` and `*big.Float` here?
I'm not familiar with those type.
Can you provide a reason?

Reply:
I don't see the point of being compatible with this when most JSON parser / encoder don't support number bigger than 64bits.

Reply:
> most JSON parser / encoder don't support number bigger than 64bits.

python:
```python
>>> json.loads('115792089237316195423570985008687907853269984665640564039457584007913129639936')
115792089237316195423570985008687907853269984665640564039457584007913129639936L
```

node:
```
> JSON.parse('115792089237316195423570985008687907853269984665640564039457584007913129639936')
1.157920892373162e+77
```

go support this with big, and probably every other language. 

We need to support all numbers if we are only using number type.

Reply:
> node:
> 
> ```
> > JSON.parse('115792089237316195423570985008687907853269984665640564039457584007913129639936')
> 1.157920892373162e+77
> ```
> go support this with big, and probably every other language.

Here the number is not the same. The precision has been lost.
> JavaScript numbers are always stored as double precision floating point numbers, following the international IEEE 754 standard. 
> Integers (numbers without a period or exponent notation) are accurate up to 15 digits.
> https://www.w3schools.com/js/js_numbers.asp

Reply:
I know, but it was just to show that other languages support parsing big numbers (even if the precision is lost, but this is something nodejs developer needs to worries about, we need to guarantee the go code handles data input well.)